config/locales/en.yml

```
@@ -473,6 +476,7 @@ en:
       content_policies:
         comment: Internal note
         description_html: You can define content policies that will be applied to all accounts from this domain and any of its subdomains.
+        limited_federation_mode_description_html: Limited federation mode description html
```

There gotta be some real text here but i don’t know what it is.

---

This was the only reference I saw doing a quick code search https://github.com/mastodon/mastodon/blob/6a5e447753095606e42cc3899887ba7fae77a0e8/app/views/admin/instances/show.html.haml#L40
I think there may be a better search with the `i18n-tasks` cli though

---

Maybe that reference should be to `admin.instances.content_policies.description_html` instead

---

I don't think so, the missing string is specifically for the limited federation mode. The reference was added in https://github.com/mastodon/mastodon/pull/17704 but without the actual string.

I think it should say something like “You are currently allowing federation with this domain.”

---

Do you want to add that as a suggestion for me to commit, or does it need a little more digging?

---

I mean the string didn't exist before #17704 and it was either never written at that point, or left out of the commit, so Gargron's original intent is only known to him. Looking at the code a bit more the string *might* be shown when a domain is not allowed, so my previous suggestion may not be correct. I'd suggest the following instead:

```suggestion
        limited_federation_mode_description_html: You are chose whether to allow federation with this domain.
```